src/test/resources/features/updatePosition.feature

```diff
@@ -0,0 +1,9 @@
+Feature: updateDepartment
```

Comment:
I think this Feature should be called: updatedPosition, because it is inside udpatePosition.feature file, or maybe the file name is wrong